demo/models.py

```diff
@@ -26,6 +31,44 @@
     ('private', "Private"),
 )
 
+# Global Streamfield definition
```

Comment:
We _could_ but as none of the code here is intended to be exemplar, I wasn't sure if we _should_. What would the benefit be?